api_confluence.py

```diff
@@ -0,0 +1,184 @@
+import os
```

Comment:
@rpappalax Wondering about the name of this file: api_confluence.py the others are called just with the API, jira.py, testrail.py.....

Reply:
yea the problem there is that confluence is already a pypi package.  When I named my module confluence, the script gets confused and thinks I'm referring to the pypi package. If we have to rename this in the end, we may need to rename the others too for consistency.  I'm open to suggestions :D

Reply:
After you verify this is working (not breaking anything), I will open a PR to change `bugz.py` to `api_bugzilla.py` and all the others as well.